src/sentry/static/sentry/app/stores/groupStore.jsx

```diff
@@ -52,17 +52,15 @@ const GroupStore = Reflux.createStore({
       itemIds.add(item.id);
     });
 
-    items.forEach((item, idx) => {
+    // See if any existing items are updated by this new set of items
+    this.items.forEach((item, idx) => {
       if (itemsById[item.id]) {
         this.items[idx] = jQuery.extend(true, {}, item, itemsById[item.id]);
-        // HACK(dcramer): work around statusDetails not being consistent
-        if (typeof itemsById[item.id].statusDetails !== undefined) {
-          this.items[idx].statusDetails = itemsById[item.id].statusDetails;
```

Comment:
I'm definitely nervous removing this, but AFAICT, this check was broken because:

* `typeof itemsById[item.id].statusDetails` always produces a string `"undefined"`, not `undefined`, so this check never succeeeded
* `statusDetails` would have been merged in the line above anyways

Aka it wasn't working anyways so I feel okay removing it. cc @dcramer